core-java-modules/core-java-conditionals/src/test/java/com/baeldung/conditionals/YieldTest.java

```diff
@@ -0,0 +1,25 @@
+package com.baeldung.conditionals;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class YieldTest {
+
+    @Test
+    void testAnalyze() {
+        assertEquals("Got a 1", YieldExamples.analyze(YieldExamples.Number.ONE));
```

Comment:
I'd probably add some static imports to make this read easier. Static import `YieldExamples` and then it will read `analyze(Number.ONE)` which is clearer.